### ISSUE TITLE
A few minor issues that are said to be resolved

### ISSUE THREAD

**Issue by relikd** [lektor/lektor#1022]
From the changelog:
```
3.3.1
Fix spastic scroll behavior when editing flow elements. #640

3.3.0
Add "<ctl>-e" hotkey shortcut to edit page. #876
Disable the "Save" button unless there are changes. #872
```

With the latest release 3.3.3 (and 3.3.2) I have the following remarks:
1. The scroll bug seems to persist. If I edit a flow-block with very long content (content that is higher than the window browser), Lektor scrolls to the top of the flow-block. I can not see what I edit at the bottom of the flow block markdown field.
2. The ctl+e hotkey does not seem to work at all (macOS). None of these work: cmd+e, ctrl+e, alt+e, fn+e and any combination with shift.
3. The save button is disabled when no changes are made. This is fine, but can the hotkey cmd+s at least open the preview? This impairs my workflow to preview a page (if I havent changed anything).

**Comment by relikd** [comment]
Adding to the list:
- `size = large` and `size = small` does not seem to work anymore. All fields have the same size regardless of that option.

**Comment by nixjdm** [comment]
The scroll bug has been re-reported as #1038. Rather than mark that as a duplicate, I think we should just track that one over there in a dedicated issue, since it's so crippling to using the admin interface.

**Comment by nixjdm** [comment]
And FWIW, I also vote for the save hotkey to always open the preview window. I also am in the habit of using that to quickly change my view, even if I hadn't actually made an edit.

**Comment by dairiki** [comment]
> 2\. The ctl+e hotkey does not seem to work at all (macOS). None of these work: cmd+e, ctrl+e, alt+e, fn+e and any combination with shift.

I think this is because when the preview `<iframe>` has the focus, the iframe gets the `keydown` events rather than the top-level page.

If I click outside of the preview iframe (so that the top-level window gets the focus), then ctl-e and ctl-g work for me.
If I click in the iframe, the ctl-e and ctl-g open Chrome's search dialogs.

I'm not sure there is a way to get keypresses in the iframe passed to the parent window.  Hmph.

**Comment by relikd** [comment]
> If I click outside of the preview iframe (so that the top-level window gets the focus), then ctl-e and ctl-g work for me.

Just tried this (macOS, Firefox) and no, even that does not work for me. Cmd+G works as you described, only if the iframe has no focus. But Cmd+E does not work at all (neither does Alt+E, Ctrl+E, Fn+E, or Shift+E, just to be sure)

**Comment by dairiki** [comment]
> Just tried this (macOS, Firefox) and no, even that does not work for me. Cmd+G works as you described, only if the iframe has no focus. But Cmd+E does not work at all (neither does Alt+E, Ctrl+E, Fn+E, or Shift+E, just to be sure)

I wonder if Cmd+E is being stolen by the window manager or the browser?

Here is a codepen which simply reports what `keydown` events it is receiving: https://codepen.io/dairiki/full/QWmvELw.
See if you can get it to report receiving a `Meta+e`.

**Comment by relikd** [comment]
Yes that works. Pressing and holding the command key generates a Meta+Meta event. But then pressing e prints the Meta+e message.
Additionally, the Meta+e functionality did work properly in a previous version.

**Comment by dairiki** [comment]
@relikd Thank you for the report. 

Well, I'm stumped 🤷.  As noted previously, Ctl+e works for me (Linux), so long as I'm careful that the iframe doesn't have focus.

The [codepen](https://codepen.io/dairiki/full/QWmvELw) reports `getPlatform() ⇒ mac` in the second line of output, right?

**Comment by relikd** [comment]
> The [codepen](https://codepen.io/dairiki/full/QWmvELw) reports `getPlatform() ⇒ mac` in the second line of output, right?

yes

**Comment by dairiki** [comment]
@relikd Aha! I was testing with `master`.  I'm not quite sure what the difference is, but I can reproduce <ctl>-e not working with 3.3.5.

**Comment by dairiki** [comment]
Okay, I think that hopefully🤞 this is all fixed in release [3.3.6](https://github.com/lektor/lektor/releases/tag/v3.3.6).  (If not please reopen.)
